conda_pack/scripts/windows/deactivate.bat

```diff
@@ -0,0 +1,46 @@
+@REM @ symbols in this file indicate that output should not be printed.
+@REM   Setting it this way allows us to not touch the user's echo setting.
+@REM   For debugging, remove the @ on the section you need to study.
+@REM
+@REM Run any activate scripts
+@REM Do this before running setlocal so that variables are cleared properly
+@IF EXIST "%CONDA_PREFIX%\etc\conda\deactivate.d" (
+    @PUSHD "%CONDA_PREFIX%\etc\conda\deactivate.d"
+    @FOR %%g in (*.bat) DO @CALL "%%g"
+    @POPD
+)
+
+@setlocal enabledelayedexpansion
+
+@REM If there's no active environment, there's nothing to do
+@IF "%CONDA_PREFIX%" == "" @GOTO skipdeactivate

```

Comment:
Shouldn't this go before the deactivate scripts, in case there's no deactivate scripts to run?

Reply:
Agreed, moved.